package.json

```diff
@@ -62,6 +62,7 @@
   },
   "private": true,
   "dependencies": {
+    "@actions/core": "^1.10.0",
```

Comment:
Is there a reason we use npm AND pnpm? (see both `package-lock.json` and `pnpm-lock.yaml`)